server/cloudformation/cloudformation.go

```diff
@@ -296,3 +314,37 @@ func (c *CustomResourceProvisioner) Handle(message *sqs.Message) error {
 
 	return nil
 }
+
+// IntValue defines an int64 type that can parse integers as strings from json.
+// It's common to use `Ref`'s inside templates, which means the value of some
+// properties could be a string or an integer.
+type IntValue int64
+
+func intValue(v int64) *IntValue {
+	i := IntValue(v)
+	return &i
+}
+
+func (i *IntValue) UnmarshalJSON(b []byte) error {
```

Comment:
This gets used when we decode the resource request into a `Request` (See the DesiredCount property of ECSServiceProperties).